packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -164,13 +163,21 @@ const renderHeaderContent = ({
 
   const liveLabelIsEnglish = liveLabel === 'LIVE';
 
-  const formattedStartTime = formatUnixTimestamp(
-    startTime,
-    'HH:mm',
+  const formattedStartTime = formatUnixTimestamp({
+    timestamp: startTime,
+    format: 'HH:mm',
     timezone,
     locale,
-    false,
-  );
+    isRelative: false,
+  });
+
+  const episodeTitle = formatUnixTimestamp({
+    timestamp: startTime,
+    format: 'D MMMM YYYY HH:mm z',
```

Comment:
Do we want to be including the time in the episode title, as we already have it in the start time above the card?

Reply:
@tochwill I've updated the format to have just the long date format `LL` that we use for timestamps on Story Promos.